api/src/test/java/com/epam/pipeline/manager/pipeline/RunScheduleManagerTest.java

```diff
@@ -16,301 +16,276 @@
+import com.epam.pipeline.test.creator.pipeline.PipelineCreatorUtils;
 import org.junit.Before;
 import org.junit.Test;
-import org.mockito.Mockito;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.test.mock.mockito.MockBean;
-import org.springframework.security.test.context.support.WithMockUser;
-import org.springframework.test.annotation.DirtiesContext;
-import org.springframework.test.context.ContextConfiguration;
-import org.springframework.transaction.annotation.Transactional;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+
+import static org.mockito.Matchers.any;
+
+import org.mockito.MockitoAnnotations;
 
 import java.util.Arrays;
 import java.util.Collections;
-import java.util.List;
+import java.util.Date;
+import java.util.Optional;
 import java.util.TimeZone;
-import java.util.stream.Collectors;
 
-import static org.junit.Assert.assertEquals;
-import static org.mockito.Matchers.any;
+import static org.mockito.Matchers.anyListOf;
+import static org.mockito.Matchers.anyLong;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.times;
 
-@DirtiesContext
-@Transactional
-@ContextConfiguration(classes = TestApplicationWithAclSecurity.class)
-public class RunScheduleManagerTest extends AbstractManagerTest {
+public class RunScheduleManagerTest {
 
     private static final Long RUN_ID = 1L;
     private static final String CRON_EXPRESSION1 = "0 0 12 * * ?";
     private static final String CRON_EXPRESSION2 = "0 15 10 ? * *";
     private static final String TEST_NAME = "TEST";
-    private static final String TEST_REPOSITORY = "///";
-    private static final TimeZone TIME_ZONE = TimeZone.getTimeZone("UTC");
-    private static final String TEST_PIPELINE_REPO_SSH = "git@test";
-    private static final String USER_OWNER = "OWNER";
+    private static final String TIME_ZONE = "Coordinated Universal Time";
+
+    private final RunConfiguration runConfiguration = ConfigurationCreatorUtils.getRunConfiguration(2L, TEST_NAME);
+    private final Pipeline testPipeline = PipelineCreatorUtils.getPipeline(RUN_ID, TEST_NAME);
+    private final PipelineRunScheduleVO testRunScheduleVO =
+            PipelineCreatorUtils.getPipelineRunScheduleVO(RunScheduledAction.PAUSE, CRON_EXPRESSION1, TIME_ZONE);
+    private final PipelineRunScheduleVO testRunScheduleVO2 =
+            PipelineCreatorUtils.getPipelineRunScheduleVO(RunScheduledAction.RESUME, CRON_EXPRESSION2, TIME_ZONE);
+    private final PipelineRunScheduleVO testRunScheduleVO3 =
+            PipelineCreatorUtils.getPipelineRunScheduleVO(RunScheduledAction.RUN, CRON_EXPRESSION2, TIME_ZONE);
+    private final RunSchedule runPauseSchedule = createRunSchedule(RUN_ID, RunScheduledAction.PAUSE, CRON_EXPRESSION1);
+    private final RunSchedule runResumeSchedule = createRunSchedule(2L, RunScheduledAction.RESUME, CRON_EXPRESSION2);
```

Comment:
It is very confusing to see `RUN_ID` used as run schedule id. Let's add two more constants for run schedule ids and use it in the class rather than using `1L` and `2L` values.

```java
private static final Long RUN_ID = 1L;
private static final Long PAUSE_SCHEDULE_ID = 1L;
private static final Long RESUME_SCHEDULE_ID = 2L;

private final RunSchedule runPauseSchedule = createRunSchedule(PAUSE_SCHEDULE_ID, RunScheduledAction.PAUSE, CRON_EXPRESSION1);
private final RunSchedule runResumeSchedule = createRunSchedule(RESUME_SCHEDULE_ID, RunScheduledAction.RESUME, CRON_EXPRESSION2);

...

testRunScheduleVO.setScheduleId(PAUSE_SCHEDULE_ID);

...

doReturn(Optional.of(runPauseSchedule))
        .when(mockRunScheduleDao).loadRunSchedule(PAUSE_SCHEDULE_ID);

# and etc
```